tools/teradata-bigquery-sql-translator/.gitignore

```diff
@@ -0,0 +1,16 @@
+!project/plugins.sbt
+*.DS_Store
+*.gz
+*.iml
+*.ipr
+*.iws
+*.log
+*.swp
+.idea
```

Comment:
do you need both .idea and .idea/*? won't just .idea/ work? Same for target/*

Reply:
You're right, they're not necessary. It was left over from the original project. Thanks for catching this, I realized that project was being excluded and I've added build.properties and plugins.sbt to the PR.